space-cli/cmd/modules/database/commands.go

```diff
@@ -10,23 +10,30 @@ import (
 func GenerateSubCommands() []*cobra.Command {
 
 	var generaterule = &cobra.Command{
-		Use:  "db-rules",
+		Use:  "db-rule [path to config file]",
 		RunE: actionGenerateDBRule,
+		Aliases: []string{"db-rules"},
+		Example: "space-cli generate db-rule config.yaml --project myproject --log-level info",
 	}
 
 	var generateconfig = &cobra.Command{
-		Use:  "db-config",
+		Use:  "db-config [path to config file]",
 		RunE: actionGenerateDBConfig,
+		Aliases: []string{"db-configs"},
+		Example: "space-cli generate db-config config.yaml --project myproject --log-level info",
 	}
 
 	var generateschema = &cobra.Command{
-		Use:  "db-schema",
+		Use:  "db-schema [path to config file]",
 		RunE: actionGenerateDBSchema,
+		Aliases: []string{"db-schemas"},
+		Example: "space-cli generate db-schema config.yaml --project myproject --log-level info",
 	}
 
 	var generatePreparedQuery = &cobra.Command{
 		Use:  "db-prepared-query",
 		RunE: actionGenerateDBPreparedQuery,
+		Example: "space-cli generate db-prepared-query config.yaml --project myproject --log-level info",
```

Comment:
You missed the Alias here

Reply:
I wasn't sure if I should add an alias here. The cli doesn't ask the user if he wants to add more prepared queries after adding one, so I thought that adding an alias for it would be unecessary

Reply:
Thats a valid point. But i think all generators behave that way. I proposed adding Alias to makenit consistent with the getter APIs